validator/nodejs/package.json

```diff
@@ -24,6 +24,6 @@
     "jasmine": "2.3.2"
   },
   "scripts": {
-    "postinstall": "/bin/sh -c \"exit 0\" 2> postinstall.DELETEME && rm postinstall.DELETEME || node postinstall-windows.js"
+    "postinstall": "node postinstall-windows.js"
```

Comment:
There are various Ubuntu / Debian distros in which this command is called node, not nodejs. E.g., see this here:
https://stackoverflow.com/questions/18130164/nodejs-vs-node-on-ubuntu-12-04
So I think this would break.